torch/testing/_internal/common_methods_invocations.py

```diff
@@ -904,6 +904,13 @@ def sample_inputs_max_min_reduction_no_dim(op_info, device, dtype, requires_grad
                                           requires_grad=requires_grad),))
     return inputs
 
+def sample_inputs_outer(op_info, device, dtype, requires_grad):
+    inputs = []
```

Comment:
This is good but you can code golf it slight by doing the following:

```
arg_a = make_tensor((S,), device, dtype, requires_grad=requires_grad)
arg_b = make_tensor((M,), device, dtype, requires_grad=requires_grad)
return (SampleInput(arg_a, args=(arg_b,),)
```

Reply:
(Let's not bother changing this PR and just land it as-is, however. The change would have a minimal impact and we're going to revise sample input functions shortly, anyway.)

Reply:
@mruberry yeah, I though about that, just didn't want to break the visual rhythm of the other sample_input_*'s :) I'll keep that on mind for the next round! btw, is there a place (slack?..) where planned changes (like revising sample inputs) are discussed or it's a strictly internal business? I'm asking because following such discussions could be helpful.